distbench_busybox.cc

```diff
@@ -154,6 +154,28 @@ absl::Status SaveResultProtoToFileBinary(const std::string &filename,
   return absl::OkStatus();
 }
 
+absl::StatusOr<int64_t> GetTestSequenceTimeout(
+    const distbench::TestSequence &test_sequence) {
+  int64_t accumulated_timeout = 0;
+  for (const auto& test : test_sequence.tests()) {
+    auto maybe_timeout = GetNamedAttributeInt64(test, "test_timeout", -1);
+    if (!maybe_timeout.ok()) {
+      return maybe_timeout;
+    }
+    if (*maybe_timeout == -1) {
+      accumulated_timeout = -1;
+    } else if (accumulated_timeout >= 0 ) {
+      accumulated_timeout += *maybe_timeout;
+    }
+  }
+
+  if (accumulated_timeout <= 0)
```

Comment:
A test sequence with no tests should probably return a timeout of 0.

Reply:
We use an empty test sequence to check if the test_sequencer is up. Having a timeout of 0 would make the RPC to always fail. Instead let's use the command line specified value (or default) so that ping, or other potential query works.